fixtures/american-dj/flat-par-cwww9.json

```
@@ -0,0 +1,215 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "FLAT PAR CWWW9",
+  "categories": ["Blinder", "Dimmer"],
+  "meta": {
+    "authors": ["Sam"],
+    "createDate": "2024-07-10",
+    "lastModifyDate": "2024-07-10"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/7054/flat_par_cwww9.pdf"
```

```suggestion
      "https://www.adj.eu/mwdownloads/download/link/id/437"
```

The cloudfront.net one works, but I think adj.eu might be more reliable in the future?